code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -461,6 +461,16 @@
 		cavity_item = null
 	..()
 
+//Heavy hits to the chest cause internal damage
+/obj/item/bodypart/chest/receive_damage(brute = 0, burn = 0, stamina = 0, updating_health = TRUE, required_status = null)
+	var/internal = 0
+	if(brute >= INTERNAL_DAMAGE_THRESHOLD && !owner.has_trait(TRAIT_INTERNALIMMUNE))
+		internal = brute / 3
+		brute -= internal
```

Comment:
Wait, it's not just explosion damage- you're nerfing ALL brute damage to the chest above a certain threshold to be only 2/3 as (immediately) damaging.